common/eventconsumption/eventindexsummarize.cpp

```diff
@@ -46,7 +46,7 @@ bool CIndexFileSummary::visitEvent(CEvent& event)
             summary.loads++;
             summary.loadedSize += event.queryNumericValue(EvAttrExpandedSize);
             summary.read += event.queryNumericValue(EvAttrReadTime);
-            summary.elapsed += event.queryNumericValue(EvAttrElapsedTime);
+            summary.elapsed += event.queryNumericValue(EvAttrExpandTime) + event.queryNumericValue(EvAttrReadTime);
```

Comment:
Will, or should, there be a follow up Jira to summarize the expand times separately? It seems like the expand time would be more meaningful than continuing to summarize the elapsed time.

Reply:
I was going to leave that to you!